internal/core/src/segcore/ConcurrentVector.h

```diff
@@ -84,6 +90,113 @@ class VectorBase {
     virtual void
     set_data_raw(ssize_t element_offset, const void* source, ssize_t element_count) = 0;
 
+    void
+    set_data_raw(ssize_t element_offset, ssize_t element_count, const DataArray* data, const FieldMeta& field_meta) {
```

Comment:
this API should be moved to ConcurrentVector.cpp

Reply:
fix